### ISSUE TITLE
Supporting QScintilla?

### ISSUE THREAD

**Issue by henux** [mkottman/lqt#10]
Hello again,

Would it be much trouble to add Lua bindings for the QScintilla component? If you can write me a short tutorial in the Wiki I would love to do it myself and contribute back to you.

Thanks.

**Comment by mkottman** [comment]
I have QScintilla in my mental TODO list :) However, I cannot guarantee when I will get to do it (but it should be in the next months...).

If you have interest to do it (it will require a lot of fiddling...), we can take the discussion to the [lqt google group](http://groups.google.com/group/lqt-bindings).

**Comment by henux** [comment]
Yes, I might be interested of doing it since I would need syntax highlighted text editors. Can you assess how many work days it might require?
